dev1/class.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+class User{
+    protected $height;
+    protected $weight;
+    public function __construct($height,$weight){
+        $this->height = $height;
+        $this->weight = $weight;
+    }
+}
+
+class Author extends User{
```

Comment:
I recommend that separate classes be in different files. It is a best practice for readability and scalability.